aixplain/utils/file_utils.py

```diff
@@ -136,3 +138,69 @@ def upload_data(
             return upload_data(file_name, content_type, content_encoding, nattempts - 1)
         else:
             raise Exception("File Uploading Error: Failure on Uploading to S3.")
+
+def _list_files_and_folders_in_s3_bucket(bucket_name, prefix) -> tuple:
+    # fetch all the files and directory in s3 and return
```

Comment:
Generate the Docstring of the method as well as the typing of the input variables